frontend/packages/kubevirt-plugin/src/components/vms/vm-environment/vm-environment-page.tsx

```diff
@@ -337,7 +362,8 @@ const VMEnvironment = withHandlePromise<VMEnvironmentProps>(
         const sourceName = getSourceName(ed);
 
         if (sourceName) {
-          const sourceDiskName = getNewDiskName(sourceName);
+          const sourceDiskName =
+            getDiskNameBySource(sourceName, vmWrapper) || getNewDiskName(sourceName);
```

Comment:
why do we have to lookup this in reverse? This could be also buggy if the same source is there twice.

Can we refactor it a bit and use some id for identification?